_posts/2024-06-20-pyconus-pyopensci-sprints.md

```diff
@@ -0,0 +1,358 @@
+a project! As such we can't expect all open source packages to be able to
+support beginner contributions in a comprehensive way.
+
+However, if you do have the bandwidth to hold a sprint that embraces, empowers and uplifts those who are newer to open source, there
+is a lot to be learned from understanding learner motivations and types. And a few good experiences might mean that a beginner decides to stick around.
+
+## Contributing vs learning
+
+The educator inside of me can't help but align my experience in Open Source with
+learner motivations. For me personally, contributing to Open Source met two of my goals
+and interests:
+
+* **Applied (project-based) learning:** I love to learn. Coding and data science are my
+  happy places. But the learning needs to be directly applicable. If it isn't, I get  bored. But moreover, if I can't see the application of the skill, I have little motivation to learn that skill!
+* **Student-directed learning:** I love to learn on my own time, following my own
+  processes that work for me. Open Source allows me to do just that (and without the
+  pressures of a specific deadline in most cases).
+
+If you read the education literature, you will find both project-based learning and student-directed learning to be
+a commonly discussed topic - especially as it relates to empowering people who are underrepresented in a space.
+
+More on that next...
+
+### The power of project-based learning
+
+[Project-based learning](https://www.wpi.edu/project-based-learning/why-pbl) is
+founded on the idea that meaningful, applied projects are
+the best way to teach a topic. This works especially well in the data science space and
+has been found to be particularly effective with underrepresented groups. In
+fact I implemented and collected data on this in a [previous project](https://www.leahwasser.com/training-diverse-communities-earth-data-science-corps)
+that I designed and ran when I was in academia.
+
+
+The idea behind project-based learning is that students select a topic they are
+personally interested in. If they are learning data science, they can find data to
+analyze and support their project. For underrepresented groups in STEM (and most
+learners), applied learning resonates because it involves a topic they care about and
+are motivated to pursue. Skills are learned in the process of achieving a meaningful
+outcome.
+
+It's similar to how you can immediately engage a diverse audience in GIS (spatial data) with Google
+Earth by asking them to zoom into the area where they live (place-based learning).
+
+As a volunteer maintainer of Stravalib, I am motivated to work on stravalib
+because I learn in a very applied way.
+
+Sure, a sprint does not have the powerful outcome of a student understanding the
+impacts of climate change on their local tribal lands. But the concept is the same:
+
+> The learning motivation comes from a meaningful outcome that a student wants or cares
+  about.
```

Comment:
`about` should be included in the above block-quote

Reply:
<img width="895" alt="Screenshot 2024-07-25 at 4 47 12 PM" src="https://github.com/user-attachments/assets/8d478467-d7c9-41e9-817c-bb17fdae0c2d">

good catch! because that entire section is indented it actually renders ok as is. but i could add the > for review readability!